smarts/core/mission_planner.py

```diff
@@ -20,35 +20,64 @@
 import logging
 import math
 import random
-from dataclasses import replace
-from typing import Optional
+from dataclasses import dataclass, replace
+from typing import List, Optional, Sequence, Tuple
 
 import numpy as np
 
 from .agent_interface import AgentBehavior
 from .coordinates import Heading, Pose
+from .lanepoints import LanePoints, LanePoint, LinkedLanePoint
```

Comment:
Using full import paths is better than relative import paths.

Reply:
I disagree with this one for two reasons:  

First, the existing pattern here (as can be seen just above the line I added) is for relative imports.  So to make that change would imply either being inconsistent within the file (which is worse, imo), or snowballing to do all files in the project.  Again, I don't have a problem with filing a new Issue to do all of these at once, but it should be part of its own, standalone PR.

Second, absolute imports reduce portability of files within a module.